pkgs/applications/audio/traverso/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit, cmake
+, alsaLib, fftw, flac, lame, libjack2, libmad, libpulseaudio
+, libsamplerate, libsndfile, libvorbis, portaudio, qtbase, wavepack
+}:
+stdenv.mkDerivation rec {
+  name = "traverso-${version}";
+  version = "0.49.5";
+
+  src = fetchgit {
+    url = "https://git.savannah.gnu.org/git/traverso.git";
+    rev = "2e215feaa9aebe104658c14d1820abdece7fb287";
+    sha256 = "1bmfaxi3f4ppk9dp8zj5lllrkmzq3s04h066j9cqsm4v27vra6bg";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ alsaLib fftw flac.dev libjack2 lame
+                  libmad libpulseaudio libsamplerate.dev libsndfile.dev libvorbis
+                  portaudio qtbase wavepack ];
+
+  cmakeFlags = [ "-DWANT_PORTAUDIO=1" "-DWANT_PULSEAUDIO=0" "-DWANT_MP3_ENCODE=1" ];
+
+  meta = with stdenv.lib; {
+    description = "Cross-platform multitrack audio recording and audio editing suite";
+    homepage = http://traverso-daw.org/;
+    license = licenses.lgpl2;
```

```suggestion
    license = with licenses; [ gpl2Plus lgpl21Plus ];
```

Most source files contains the header for GPL-2+
```
Traverso is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.
```

But some 3rd party contains a header designing LGPL-2.1+ such as http://git.savannah.nongnu.org/cgit/traverso.git/tree/src/3rdparty/slv2/slv2/lv2.h

---

Was not aware of the ability to provide a list of licenses. That's cool! Fixed.